packages/edit-site/src/components/page-pages/style.scss

```diff
@@ -43,7 +43,7 @@
 	overflow: hidden;
 	height: 100%;
 	width: 100%;
-	border-radius: 3px 3px 0 0;
+	border-radius: 3px;
```

Comment:
Should we make this 4px?

That would be consistent with other UI's. I don't think we use 3px elsewhere. It would also fix this small discrepancy in the corners, where the background bleeds through between the image and the focus ring:

<img width="229" alt="Screenshot 2024-07-01 at 11 22 39" src="https://github.com/WordPress/gutenberg/assets/846565/92329a06-642d-403e-8d6b-615d8c3eb8f1">